docs/guide/components/camel-consul.adoc

```diff
@@ -0,0 +1,4 @@
+### camel-consul
+
+The https://github.com/apache/camel/blob/master/components/camel-consul/src/main/docs/consul-component.adoc[consul,window=_blank] component allows you to work with Consul a distributed highly available datacenter-aware service discovery and configuration system.
```

Comment:
Instead of referencing the master branch, we usually refer to the release tag that we're targeting with `camel-{camel-version}`.

Reply:
Yes, I actually know this in between, but this PR comes from times when I did not know.